guides/_subsections/view-targets-with-topology.md

```diff
@@ -0,0 +1,112 @@
+## [Use Topology View](#use-topology-view)
+
+**Topology View** provides a visual presentation of the all discovered JVM applications and all resources that are associated with them. It also allows users to perform actions on one or multiple targets.
+
+{% include_relative _subsections/common/navigate-to-topology.md %}
+
+### [View all target JVMs](#view-all-target-jvms)
+
+<ol>
+    {% capture topology-graphview-guide-text %}
+    By default, an interactive <b>Graph View</b> of discovered target JVMs (<i>nodes</i>) are shown nested their associated groups (<i>surrounding lines</i>), for example, Pod, or Realm (i.e. discovery platform). 
+    <br><br>
+
+    You can drag target nodes/groups or use the bottom control bar to adjust the graph, for example, zoom out or fit all nodes into view. A toolbar is also available to allow further customization:
+
+    <ol>
+        <li>
+            <b>Display Options:</b> adjust how the nodes and groups are displayed, for example, whether to show connection URL.
+        </li>
+        <li>
+            <b>Filters:</b> determine which targets or groups to show.
+        </li>
+        <li>
+            <b>Search bar:</b> find a target using <b>Match Expression</b>. The matched targets will be highlighted.
+        </li>
+    </ol>
+    {% endcapture %}
+    {% capture topology-listview-guide-text %}
+    Topology View also supports <b>List View</b> mode, where your JVM targets and their groups are shown as expandable rows. 
+    <br><br>
+    Click the List icon on the toolbar to switch to <b>List View</b>. Expand each row to see nested groups or targets. All above features of the toolbar can also be used to customize your view.
+    {% endcapture %}
+
+    {% include howto_step.html
+      summary="View JVM applications with Graph View"
+      image-name="2.3.0/topology-1.png"
+      caption="View JVM applications with topology Graph view."
+      text=topology-graphview-guide-text
+    %}
+    {% include howto_step.html
+      summary="View JVM applications with List View"
+      image-name="2.3.0/topology-2.png"
+      caption="View JVM applications with Topology List view."
+      text=topology-listview-guide-text
+    %}
+</ol>
+
+### [View JVM details and resources](#view-jvm-details-and-resources)
+<ol>
+
+    {% capture sidebar-detail-guide-text %}
+    In <b>Graph View</b>, select a target JVM node to open the drawer panel that shows its details, for example, connection URL, labels and annotations. Click <i>Show more</i> to the JVM’s Mbean details.
```

Comment:
"Click Show More to view additional JVM details", I don't know if MBeans need to be mentioned here. Users might not know what that is, or it may not be the only source of information we put here in the future.